src/common.ts

```diff
@@ -34,8 +34,8 @@ export async function sendRequest<T>(
 	const data = await response.json();
 
 	if (!response.ok) {
-		throw new ApiError(data);
+		throw new ApiError(data as ErrorResponseData);
 	}
 
-	return data;
+	return data as T;
```

Comment:
dont need casting, because return type defined here:
https://github.com/zhelvis/unisender-ts/blob/8f9fed2386055cdf2e4aea32e5bfbbf3083fd982/src/common.ts#L23